tools/deployment-cli-tools/ch_cli_tools/config/migration.json

```diff
@@ -0,0 +1,46 @@
+{
+    "deprecated": [
```

Comment:
it would be useful to have separate files to add other migrations

Reply:
discussed with @filippomc change this a folder and take the migrations in order, similar to django.